doc/source/ray-observability/user-guides/ray-profiling.rst

```diff
@@ -0,0 +1,83 @@
+.. _ray-profiling:
+
+Profiling on Ray
+================
+Profiling is one of the most important debugging tools to diagnose performance, out of memory, hanging, or other application issues as desribed `here <https://docs.ray.io/en/master/ray-observability/user-guides/profiling.html>`_. For better experience, Ray is natively supporting profilers to be run for task, actor or job in distributed cases.
+
+
```

Comment:
Regarding the main summary comment in favor of docs consolidation, I would remove this repeated information.